### ISSUE TITLE
System.UriFormatException: 'Invalid URI: The hostname could not be parsed.'

### ISSUE THREAD

**Issue by Ganesh-Ponipireddy** [EPPlusSoftware/EPPlus#725]
We are getting this exception while reading data when the cell value ends with `@` symbol. 
<img width="166" alt="image" src="https://user-images.githubusercontent.com/48117290/195371491-ec6d84dc-fb44-4682-877f-f737354d927b.png">

sample code: the code is in vb.net
`function Readexcel(file As HttpPostedFileBase)` 
`Dim workbook As New ExcelPackage`
  `workbook.Load(file.InputStream)`
`dim obj  = workbook.GetAsByteArray() ' Exception at this step`
`end function`

**Comment by JanKallman** [comment]
If this cells a hyperlink? Can you provide a sample workbook.

**Comment by Ganesh-Ponipireddy** [comment]
> If this cells a hyperlink? Can you provide a sample workbook.

No, those are not hyperlinks, please find the attached workbook for ref[TestWorkBook.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/9779677/TestWorkBook.xlsx)

**Comment by JanKallman** [comment]
This works when I try it...
```C#
        [TestMethod]
        public void i725()
        {
            var stream = new FileStream(_testInputPathOptional + "TestWorkbook.xlsx",FileMode.Open);
            using(var p = new ExcelPackage())
            {
                p.Load(stream);
                var ba = p.GetAsByteArray();
            }
        }
```
Can you provide a reproducible test? Which environment and regional setting are you using.

**Comment by Ganesh-Ponipireddy** [comment]
[TestData.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/9836831/TestData.xlsx)
@JanKallman please find the file, In which I'm able to reproduce.

**Comment by JanKallman** [comment]
Fixed in EPPlus 6.1
